subsys/net/ip/net_context.c

```diff
@@ -735,15 +735,15 @@ static inline int send_syn_ack(struct net_context *context,
 }
 
 static inline int send_ack(struct net_context *context,
```

Comment:
Even per my, usually strict criteria of "don't make unrelated changes", un-inlining it would fit with this patch (like, we call it from more places - don't want to inline any longer). So, will update this patch.